### PR TITLE
minor refactor in the code

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Just minor refactors are done and one new file for utilities notifications are made

### CODE REVIEW

File: src/fireBase.tsx [hunk 1]
```diff
@@ -27,19 +19,19 @@ const firebaseConfig: FireBaseConfig = {
   appId: "1:58190409157:web:4144751306c1297a98afac",
   measurementId: "G-NEFKXBHXXC",
 };
```
Comment (prachi-sahani) [inline src/fireBase.tsx:21]: Do not expose your firebase(project) credentials. Instead store them as variables in `.env` file

**Review by dev-prabhat — APPROVED** [review]
Looks good to  me
Good to merge

**Review by prachi-sahani — COMMENTED** [review]
Overall the changes look fine
Good to merge

Suggestion - consider formatting your code

File: src/Utilities/Notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+type ToastStyles={
+   position:string,
+  autoClose: number,
```
Comment (vashnavichauhan1825) [inline src/Utilities/Notifications.tsx:5]: appreciation: you separated this code!

File: src/fireBase.tsx (hunk 1 above)
Comment (Dev-AkshatGupta) [reply src/fireBase.tsx:21]: I agree to you Its just that I tried it and it started showing me error then I thought of pushing code this way only

File: src/Redux/Reducers/AuthSlice.tsx [hunk 3]
```diff
@@ -1,68 +1,75 @@
...
-})
-export default authSlice.reducer;
\ No newline at end of file
+const initialState: InitialState = {
+  currentUser: {},
+  loading: false,
+};
+const authSlice = createSlice({
+  name: "auth",
+  initialState,
```
Comment (vashnavichauhan1825) [inline src/Redux/Reducers/AuthSlice.tsx:57]: appreciation: your code is so clean and readable , everything looks good

**Review by vashnavichauhan1825 — APPROVED** [review]

File: src/Redux/Reducers/AuthSlice.tsx (hunk 3 above)
Comment (Dev-AkshatGupta) [reply src/Redux/Reducers/AuthSlice.tsx:57]: Thanks for reviewing the code

**Pull request merged**